content/how-to/dedicated-hosting/create-new-user-on-active-directory-domain/index.md

```diff
@@ -19,33 +19,36 @@ Active Directory user.
 1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="newuser3" src="/support/how-to/create-new-user-on-active-directory-domain/newuser3.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+2. In the top navigation bar, choose **Supprt** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="newuser4" src="/support/how-to/create-new-user-on-active-directory-domain/newuser4.png">
 
-5. From the drop-down menu, select **Create New User on Active Directory Domain**.
+3. From the Common Request menu select **Create New User on Active domain directory** as shown in following image.
 
-   {{<image src="newuser1.png" alt="" title="">}}
+   <img width="811" alt="newuser5" src="/support/how-to/create-new-user-on-active-directory-domain/newuser5.png">
```

Comment:
fix the alt tag here

Reply:
select required option from most commonly used template